modules/light-clients/08-wasm/testing/wasm_endpoint.go

```diff
@@ -0,0 +1,42 @@
+package wasmtesting
+
+import (
+	"github.com/stretchr/testify/require"
+
+	types "github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
+	clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+	ibctesting "github.com/cosmos/ibc-go/v7/testing"
+)
+
+type WasmEndpoint struct {
+	*ibctesting.Endpoint
+}
```

Comment:
love this!

Reply:
Big +

mega nit: should we add a basic godoc?